modules/099-ceph-csi/crds/cephcsi.yaml

```
@@ -0,0 +1,131 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: cephcsi.deckhouse.io
+  labels:
+    heritage: deckhouse
+    module: ceph-csi
+    app: ceph-csi
+spec:
+  group: deckhouse.io
+  scope: Cluster
+  names:
+    plural: cephcsi
+    kind: CephCSI
+  preserveUnknownFields: false
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema: &schema
+        openAPIV3Schema:
+          type: object
+          description: |
+            Ceph cluster connection parameters and StorageClasses configuration
```

```suggestion
            Ceph cluster connection parameters and StorageClasses configuration.
```
and everywhere in the doc.

---

Done.